tasks/src/text-to-video/data.ts

```diff
@@ -0,0 +1,92 @@
+			id:          "UCF101",
+		},
+        {
+			description: "Kinetics Human Action Video Dataset is a large-scale, high-quality dataset for human action recognition in videos. The dataset consists of around 500,000 video clips covering 600 human action classes with at least 600 video clips for each action class. Each video clip lasts around 10 seconds and is labeled with a single action class. The videos are collected from YouTube.",
+			id:          "Kinetics",
+		},
+        {
+			description: "The 20BN-SOMETHING-SOMETHING V2 dataset is a large collection of labeled video clips that show humans performing pre-defined basic actions with everyday objects.  It contains 220,847 videos, with 168,913 in the training set, 24,777 in the validation set and 27,157 in the test set. There are 174 labels.",
+			id:          "Something-Something V2",
+		},
+        {
+			description: "CelebV-Text comprises 70,000 in-the-wild face video clips with diverse visual content, each paired with 20 texts generated using the proposed semi-automatic text generation strategy. The provided texts describes both static and dynamic attributes precisely.",
+			id:          "CelebV-Text",
+		},
+        {
+			description: "This dataset consists of 10.7 million pairs of text-video pairs (52K video hours) and contains a fair amount of noisy samples with irrelevant video descriptions",
+			id:          "WebVid",
+		},
+        {
+			description: "This dataset consists of 136M video clips with captions that describe complex tasks with step-by-step information on how to perform 23k activities from domains such as cooking, hand crafting, personal care, gardening or fitness. It has 136M video clips with captions sourced from 1.2M Youtube videos (15 years of video)",
+			id:          "Howto100M",
+		},
+        {
+			description: "This dataset focuses on the event localization task such that the captions of videos describe the relative location of objects and actions in detail.",
+			id:          "QuerYD",
+		},
+        {
+			description: "The Distinct Describable Moments (DiDeMo) dataset is one of the largest and most diverse datasets for the temporal localization of events in videos given natural language descriptions. The videos are collected from Flickr and each video is trimmed to a maximum of 30 seconds.",
+			id:          "DiDeMo",
+		},
+    ],
+	demo:     {
+		inputs:  [],
+		outputs: [],
+	},
+	isPlaceholder: true,
+	metrics:       [],
+	models:        [
+        {
+			description: "Text-to-Video Generation without Text-Video Data",
+			id:          "Make-A-Video",
+		},
+        {
+			description: "Diffusion over Diffusion for eXtremely Long Video Generation",
+			id:          "NUWA-XL",
+		},
+        {
+			description: "A multi-modal AI system that can generate novel videos with text, images, or video clips",
+			id:          "Runway Gen2",
```

Comment:
The model IDs need to be valid hf.co/model repos

Reply:
Modified it. This reduced the number of available models to just 2 though. Earlier I researched a couple of model that do text2video, but not all are available on the repo